syncthing/lure.sh

```diff
@@ -8,9 +8,12 @@ license=('MPL-2.0')
 provides=('syncthing')
 conflicts=('syncthing')
 deps_amd64=('glibc' 'strelaysrv')
+deps_amd64_debian=('libc6')
 build_deps=('git' 'go')
+build_deps_debian=('git' 'golang')
+
 sources_amd64=("https://github.com/${name}/${name}/releases/download/v${version}/${name}-linux-amd64-v${version}.tar.gz")
-checksums_amd64=('f91066b8588c02b5ff77aa801cc4e0319f3467325bdd75326aebccffb6495698')
+checksums=('f91066b8588c02b5ff77aa801cc4e0319f3467325bdd75326aebccffb6495698')
```

Comment:
Please change this to `checksums_amd64`

```suggestion
checksums_amd64=('f91066b8588c02b5ff77aa801cc4e0319f3467325bdd75326aebccffb6495698')
```